src/ServiceMonitor/IISConfigUtil.cpp

```diff
@@ -54,17 +55,16 @@ IISConfigUtil::~IISConfigUtil()
     }
 }
 
-BOOL IISConfigUtil::FilterEnv(unordered_map<wstring, LPTSTR> filter, LPTSTR strEnvName, LPTSTR strEnvValue)
+BOOL IISConfigUtil::FilterEnv(const unordered_map<wstring, LPTSTR>& filter, LPCTSTR strEnvName, LPCTSTR strEnvValue)
 {
     LPTSTR   strFilterValue;
     wstring strFilterName;
     _ASSERT(strEnvName != NULL);
     _ASSERT(strEnvValue != NULL);
 
-    CharUpper(strEnvName);
     strFilterName = strEnvName;
-    auto value = filter.find(strFilterName);
-    if (value == filter.end())
+    auto value = (filter).find(strFilterName);
+    if (value == (filter).end())
     {
         return FALSE;
     }

```

Comment:
Comment for line 75: what does that comment mean:

"don't need to match value or value match"